starter-code/main.js

```diff
@@ -0,0 +1,292 @@
+const MongoDB = require('mongodb');
+const mongoClient = MongoDB.MongoClient;
+const clear = require('clear');
+const readline = require('readline');
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+const url = `mongodb://localhost:27017/crunchbase`;
+
+mongoClient.connect(url, (error, db) => {
+  if (error) {
+    console.log('Error trying to connect to the Database');
+    console.log(error);
+  } else {
+    console.log('Connection established correctly!! 😬');
+
+    function mainMenu() {
+      clear();
+      printMenu();
+      rl.question('Type an option: ', (option) => {
+        switch (option) {
+          case "1":
+            db.collection('companies').find({}, {
```

Comment:
you should have coded everything in one line in the query like this:
	**db.collection('companies').find({}, {name: 1, _id: 0}).toArray((error, result) => {**